modules/observability-key-performance-metrics.adoc

```diff
@@ -127,17 +131,15 @@ $ oc get routes -n openshift-console console -o jsonpath='{.status.ingress[0].ho
 
 |===
 
-[id="recommendations-for-storage-of-metrics"]
+[id="recommendations-for-storage-of-metrics_{context}"]
 == Recommendations for storage of metrics
 
-Out of the box, Prometheus does not back up saved metrics with persistent storage.
-If you restart the Prometheus pods, all metrics data are lost.
-You should configure the monitoring stack to use the back-end storage that is available on the platform.
-To meet the high IO demands of Prometheus you should use local storage.
-
-For Telco core clusters, you can use the Local Storage Operator for persistent storage for Prometheus.
+By default, Prometheus does not back up saved metrics with persistent storage. 
+If you restart the Prometheus pods, all metrics data are lost. 
+You must configure the monitoring stack to use the back-end storage that is available on the platform. 
+To meet the high IO demands of Prometheus, use local storage.
 
-{odf-first}, which deploys a ceph cluster for block, file, and object storage, is also a suitable candidate for a Telco core cluster.
+For smaller clusters, you can use the Local Storage Operator for persistent storage for Prometheus. {odf-first}, which deploys a ceph cluster for block, file, and object storage, is suitable for larger clusters.
```

Comment:
Is there any guidance on what constitutes a larger vs smaller clusters?

Reply:
@rfisher001 Could you please take a look?